src/redux/services/schema.ts

```diff
@@ -75,3 +83,35 @@ export function loadCustomSchema(schemaPath: string, resourceKind: string): any
     schemaCache.set(resourceKind, undefined);
   }
 }
+
+export function extractSchema(crd: any, versionName: string) {
+  const versions: any[] = crd.spec.versions;
+  const version = versions.find((v: any) => v.name === versionName);
+  const schema = version?.schema?.openAPIV3Schema ? version.schema.openAPIV3Schema : undefined;
```

Comment:
This looks redundant, you can simply do `const schema = version?.schema?.openAPIV3Schema;`